src/Avalonia.Themes.Fluent/ComboBoxItem.xaml

```diff
@@ -1,9 +1,25 @@
 <Styles xmlns="https://github.com/avaloniaui">
+  <Design.PreviewWith>
+    <Border Padding="20">
+      <StackPanel Spacing="10">
+        <ComboBox>
+          <ComboBoxItem>Item 1</ComboBoxItem>
+          <ComboBoxItem>Item 2</ComboBoxItem>
+        </ComboBox>
+
+        <ComboBox IsEnabled="False">
+          <ComboBoxItem>Item 1</ComboBoxItem>
+          <ComboBoxItem>Item 2</ComboBoxItem>
+        </ComboBox>
+      </StackPanel>
+    </Border>
+  </Design.PreviewWith>
+
   <Style Selector="ComboBoxItem">
-    <Setter Property="Background" Value="Transparent"/>
-    <Setter Property="Padding" Value="2"/>
-    <Setter Property="HorizontalAlignment" Value="Stretch"/>
-    <Setter Property="HorizontalContentAlignment" Value="Left"/>
+    <Setter Property="TextBlock.Foreground" Value="{DynamicResource ComboBoxItemForeground}" />
+    <Setter Property="Background" Value="{DynamicResource ComboBoxItemBackground}" />
+    <Setter Property="Padding" Value="{DynamicResource ComboBoxItemThemePadding}" />
```

Comment:
Interesting moment. I noticed some margin or border between ComboBoxItems in WinUI, but couldn't found it in the [style](https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/ComboBox/ComboBox_themeresources.xaml#L705). With devtools I found that there is actually border, which is sets in generic.xaml and is not overridden by WinUI. Resources (they are missed in Avalonia in master):
```xaml
<StaticResource x:Key="ComboBoxItemRevealBorderBrush" ResourceKey="SystemControlTransparentRevealBorderBrush" />
<Thickness x:Key="ComboBoxItemRevealBorderThemeThickness">1</Thickness>
```
```suggestion
    <Setter Property="Padding" Value="{DynamicResource ComboBoxItemThemePadding}" />
    <Setter Property="BorderBrush" Value="{ThemeResource ComboBoxItemRevealBorderBrush}" />
    <Setter Property="BorderThickness" Value="{ThemeResource ComboBoxItemRevealBorderThemeThickness}" />
```

Reply:
For another controls we also should look at both styles - from generic.xaml and from WinUI, to combine setters from them.